iceoryx_posh/source/runtime/ipc_interface_user.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2019, 2020 by Robert Bosch GmbH, Apex.AI Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "iceoryx_posh/internal/runtime/ipc_interface_user.hpp"
+
+namespace iox
+{
+namespace runtime
+{
+IpcInterfaceUser::IpcInterfaceUser(const ProcessName_t& name,
```

Comment:
Would maybe `IpcInterfaceClient` more fit here due to the fact that it is creating an IPC Channel client?

Reply:
Technically you're right. However, I'm not sure if this will make it harder to distinguish between the `ClientPortUser`, the RPC mechanism we will offer in the near future. I'd stick with the current name and rename it later if sensible.